scripts/perf/cascadelakex/clx-metrics.json

```diff
@@ -163,31 +163,31 @@
     },
     {
         "BriefDescription": "Ratio of number of code read requests missing last level core cache (includes demand w/ prefetches) to the total number of completed instructions",
-        "MetricExpr": "cha@UNC_CHA_TOR_INSERTS.IA_MISS\\,config1\\=0x12CC0233@ / INST_RETIRED.ANY",
+        "MetricExpr": "llc_data_read_mpi_demand_plus_prefetch",
         "MetricName": "llc_code_read_mpi_demand_plus_prefetch",
         "ScaleUnit": "1per_instr"
     },
     {
         "BriefDescription": "Average latency of a last level cache (LLC) demand and prefetch data read miss (read memory access) in nano seconds",
-        "MetricExpr": "1e9 * (cha@UNC_CHA_TOR_OCCUPANCY.IA_MISS\\,config1\\=0x40433@ / cha@UNC_CHA_TOR_INSERTS.IA_MISS\\,config1\\=0x40433@) / (UNC_CHA_CLOCKTICKS / (source_count(UNC_CHA_CLOCKTICKS) * #num_packages)) * duration_time",
+        "MetricExpr": "llc_data_read_demand_plus_prefetch_miss_latency_for_local_requests",
         "MetricName": "llc_data_read_demand_plus_prefetch_miss_latency",
         "ScaleUnit": "1ns"
     },
     {
         "BriefDescription": "Average latency of a last level cache (LLC) demand and prefetch data read miss (read memory access) addressed to local memory in nano seconds",
-        "MetricExpr": "1e9 * (cha@UNC_CHA_TOR_OCCUPANCY.IA_MISS\\,config1\\=0x40432@ / cha@UNC_CHA_TOR_INSERTS.IA_MISS\\,config1\\=0x40432@) / (UNC_CHA_CLOCKTICKS / (source_count(UNC_CHA_CLOCKTICKS) * #num_packages)) * duration_time",
+        "MetricExpr": "llc_data_read_demand_plus_prefetch_miss_latency_for_remote_requests",
         "MetricName": "llc_data_read_demand_plus_prefetch_miss_latency_for_local_requests",
```

Comment:
This looks like the same issue as on skylakex which is fixed in a later commit - local requests metric is aliased to remote requests. Could we make the fix on all architectures and squash the fix commit into this one so that we don't need to spot bugs then check for later fixes?

Reply:
@captain5050, this error turns out to be an issue caused by the mixed commits in this PR. @edwarddavidbaker just helped me updated this PR to one squashed commit. Hopefully, this makes it cleaner and easier to review.